app/visits/edit/template.hbs

```diff
@@ -226,6 +227,7 @@
                 </tr>
                 {{#each model.vitals as |vital|}}
                   <tr>
+                    <td>{{vital.modifiedBy}}</td>
```

Comment:
I am going to merge this in with the vitals using the modifiedBy field.  I think that field is sufficient